cura/MultiplyObjectsJob.py

```diff
@@ -56,19 +58,33 @@ def run(self) -> None:
 
             for _ in range(self._count):
                 new_node = copy.deepcopy(node)
-
                 # Same build plate
                 build_plate_number = current_node.callDecoration("getBuildPlateNumber")
                 new_node.callDecoration("setBuildPlateNumber", build_plate_number)
                 for child in new_node.getChildren():
                     child.callDecoration("setBuildPlateNumber", build_plate_number)
-
-                nodes.append(new_node)
+                if not current_node.getParent().callDecoration("isSliceable"):
+                    nodes.append(new_node)
+                else:
+                    # The node we're trying to place has another node that is sliceable as a parent.
+                    # As such, we shouldn't arrange it (but it should be added to the scene!)
+                    nodes_to_add_without_arrange.append(new_node)
+                    new_node.setParent(current_node.getParent())
 
         found_solution_for_all = True
```

Comment:
This variable seems to always be True now. Shouldn't it change based on the outcome of the `createGroupedOperationForArrange()`? 
If it doesn't, then the message `no_full_solution_message` later on can be removed.